docs/src/examples/components/Dropdown/State/DropdownExampleLoading.shorthand.tsx

```diff
@@ -1,17 +1,22 @@
+import { useBooleanKnob } from '@stardust-ui/docs-components'
 import { Dropdown } from '@stardust-ui/react'
 import * as React from 'react'
 
 const inputItems = ['Bruce Wayne', 'Natasha Romanoff', 'Steven Strange', 'Alfred Pennyworth']
 
-const DropdownExampleLoading: React.FC<{ knobs: { loading: boolean } }> = ({ knobs }) => (
-  <Dropdown
-    loading={knobs.loading}
-    loadingMessage="Loading..."
-    multiple
-    search
-    items={inputItems}
-    placeholder="Start typing a name"
-  />
-)
+const DropdownExampleLoading = () => {
+  const [loading] = useBooleanKnob({ name: 'loading', initialValue: true })
```

Comment:
for this specific example, just to make it user friendly in terms of user being able to notice effect, I might suggest to introduce `open` knob in addition that will be triggered by default. This will make an effect of `loading` apparent to the client